diff --git a/include/rex/chrono/chrono.h b/include/rex/chrono/chrono.h
index 7b140f9..6fc88a0 100644
--- a/include/rex/chrono/chrono.h
+++ b/include/rex/chrono/chrono.h
@@ -116,9 +116,9 @@ using XSystemClock = detail::NtSystemClock<detail::Domain::Guest>;

 namespace std::chrono {

-#ifdef __APPLE__
-// Apple libc++ does not expose clock_time_conversion or clock_cast.
+#if defined(__APPLE__) || REX_PLATFORM_ANDROID
+// Apple and Android libc++ do not expose clock_time_conversion or clock_cast.
 template <class, class>
 struct clock_time_conversion {};

 template <class DestClock, class SourceClock, class Duration>
